console-subscriber/src/init.rs

```diff
@@ -0,0 +1,81 @@
+use crate::TasksLayer;
+use std::thread;
+use tokio::runtime;
+use tracing_core::{subscriber::Subscriber, Dispatch};
+use tracing_subscriber::{fmt, layer::Layered, prelude::*, EnvFilter, Layer, Registry};
+
+type ConsoleSubscriberLayer =
+    Layered<TasksLayer, Layered<EnvFilter, Layered<fmt::Layer<Registry>, Registry>>>;
+
+/// Starts the console subscriber server on its own thread
+///
+/// This function represents the easiest way to get started using
+/// tokio-console.
+///
+/// Please note that this registers the exclusive tracing subscriber
+/// for your application. If you need to add additional layers to this
+/// subscriber, see [`init_with_layer`].
```

Comment:
couple minor suggestionshere: 
- i would use the terminology "global default subscriber", as it's still possible for the default subscriber to be temporarily overridden.
- in general, we tend to prefer prefixing notes like this with `**Note**:` in documentation (but it's not a big deal).
- it could also be nice to link to the `tracing` docs here to make things easier for users who aren't familiar with `tracing`.

with all of that said, what about something like this?
```suggestion
/// **Note**: this function sets the [default `tracing` subscriber][default]
/// for your application. If you need to add additional layers to this
/// subscriber, see [`init_with_layer`].
///
/// [default]: https://docs.rs/tracing/0.1.26/tracing/dispatcher/index.html#setting-the-default-subscriber
```